members/views/unionMembersView.py

```diff
@@ -0,0 +1,33 @@
+from django.utils import timezone
+from django.shortcuts import render
+from django.contrib.auth.decorators import login_required, user_passes_test
+from members.models import Union
+from members.utils.user import has_user
+
+
+@login_required
+@user_passes_test(has_user, "/admin_signup/")
+def unionMembersView(request, union_id):
+    # get chosen union
+    union = Union.objects.filter(pk=union_id)
```

Comment:
Når du i et view ved at du kun vil have fat i et objekt og ikke en liste kan du bruge
[get_object_or_404](https://docs.djangoproject.com/en/3.0/topics/http/shortcuts/#id2). 

Så slipper du både for at tage noget ud af listen og fixer samtidigt hvis det ID ikke findes.

Reply:
Nice, sparer nogle [0] et par steder.